integration_test/validate_metadata_test.go

```diff
@@ -28,6 +31,27 @@ func TestValidateMetadataOfThirdPartyApps(t *testing.T) {
 	}
 }
 
+func TestMissingMetadataForThirdPartyApps(t *testing.T) {
```

Comment:
[Optional]

it might be more explicit to name this `TestRequireMetadataForAllThirdPartyApps`

Reply:
Changed in: [8e4de5c](https://github.com/GoogleCloudPlatform/ops-agent/pull/793/commits/8e4de5c7b435508e229c3e0ab8dbda8c5a3b67b7)